src/features/read/readSlice.ts

```diff
@@ -224,7 +267,7 @@ export function selectDifficulty(state: RootState): Difficulty {
   const book = selectBook(state);
   const answers = state.read.answers;
 
-  if (!book || answers.length < NUMBER_CORRECT_TO_ADVANCE) {
+  if (!book || answers.length < NUMBER_CORRECT_TO_ADVANCE || book?.fields.type === "pre-test" || book?.fields.type === "post-test") {
```

Comment:
Since you have had to do this type of check a couple of times, I think it would be worth adding a method to the book model and using it instead.

Something like:

```ts
function isAssessment(book: IBook) : boolean {
  return book.fields.type === "pre-test" || book.fields.type === "post-test"
}
```